src/main/scala/dcos/metronome/JobApplicationLoader.scala

```diff
@@ -36,6 +37,11 @@ class JobApplicationLoader extends ApplicationLoader with StrictLogging {
 
     jobComponents.metricsModule.start(jobComponents.actorSystem)
 
+    val startedAt = System.currentTimeMillis()
```

Comment:
Maybe move `startedAt` into class private val in case `load()` is called multiple times?

Reply:
I think it should not be called multiple times, we would have more problems if it was but done ✅